python/cuvs/cuvs/neighbors/prefilters/prefilters.pyx

```diff
@@ -0,0 +1,97 @@
+#
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
please rename `prefilters` -> `filters` everywhere.

Reply:
All done. To double check, I suppose you might also recommend I rename `cuvsPrefilter` to `cuvsFilter.` pls refer to this commit: https://github.com/rapidsai/cuvs/pull/174/commits/f451ec6fae80fb8749e71825774e481c2edd80c4